src/core/lib/iomgr/cfstream_handle.cc

```diff
@@ -47,7 +48,7 @@ void CFStreamHandle::Release(void* info) {
 
 CFStreamHandle* CFStreamHandle::CreateStreamHandle(
```

Comment:
What's the value of this factory function?  It seems to just pass its arguments to the ctor, so why not just have callers construct it directly?